### PR TITLE
Equipments - 21030275

### PR DESCRIPTION
**Pull request by MirhasVida**

### CODE REVIEW

File: Translations/ko-KR/quest.json [hunk 1]
```diff
@@ -164,51 +164,51 @@
...
+		"desc": "이호 잠수함 3척 이상으로 이루어진 잠수함대를 편성하라!"
 	},
 	"128": {
 		"code": "A28",
 		"name_jp": "「第六戦隊」を編成せよ！",
-		"name": "Form the 6th Cruiser Division",
+		"name": "「제6전대」를 편성하라!",
 		"desc_jp": "「古鷹」「加古」「青葉」「衣笠」を基幹とした第六戦隊を編成せよ！",
-		"desc": "Have Furutaka (古鷹), Kako (加古), Aoba (青葉) and Kinugasa (衣笠) in your main fleet."
+		"desc": ""카코", "후루타카", "아오바", "키누가사"로 이루어진 제6전대를 편성하라!"
```
Comment (Jebzou) [inline Translations/ko-KR/quest.json]: You need to escape the double quotes " with a backslash \
e.g. : ```"\"카코\", \"후루타카\", \"아오바\", \"키누가사\"로 이루어진 제6전대를 편성하라!"```

**Comment by MirhasVida** [comment]
Got it, I'll edit it asap

File: Translations/ko-KR/quest.json [hunk 2]
```diff
@@ -248,51 +248,51 @@
...
+		"desc": "묘코, 나치, 하구로 를 중심으로 하는 제5전대를 편성하라!"
 	},
 	"138": {
 		"code": "A36",
 		"name_jp": "新編「第二航空戦隊」を編成せよ！",
-		"name": "Form the New 2nd Carrier Division!",
+		"name": "새로운 「제2항공전대」를 편성하라!",
 		"desc_jp": "「飛龍改二」(※旗艦)及び「蒼龍」を基幹とした二航戦(要駆逐艦2隻)を編成せよ！",
-		"desc": "Have Hiryuu Kai2 (飛龍改二) as the flagship, Souryuu (蒼龍) and 2 DD in your main fleet."
+		"desc": "히류 改2를 기함으로 소류를 기본으로 한 2항전(구축함 2척 필요)을 편성하라!
```
Comment (myangelkamikaze) [inline Translations/ko-KR/quest.json]: Missing " at the end here.

**Comment by MirhasVida** [comment]
Solved!

**Pull request merged**